spec/features/courses/about_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+feature 'About course', type: :feature do
+  let(:course) do
+    jsonapi(
+      :course,
+      provider: jsonapi(:provider, provider_code: 'A0')
+    ).render
+  end
+  let(:course_attributes) { course[:data][:attributes] }
+
+  before do
+    stub_omniauth
+    stub_api_v2_request(
+      "/providers/AO/courses/#{course_attributes[:course_code]}?include=site_statuses.site,provider.sites,accrediting_provider",
+      course
+    )
+  end
+
+  scenario 'viewing the about courses page' do
+    visit about_provider_course_path('AO', course_attributes[:course_code])
+
+    expect(find('.govuk-caption-xl')).to have_content(
```

Comment:
Can we add a SitePrism page while we're here pls?

Reply:
Done